app/models/borrow_direct_requests.rb

```diff
@@ -39,6 +39,17 @@ def key
       request_number
     end
 
+    def sort_key(sort)
+      case sort
+      when :title
+        title
+      when :date
+        ::Request::END_OF_DAYS.strftime('%FT%T')
```

Comment:
```suggestion
        [::Request::END_OF_DAYS.strftime('%FT%T'), title].join('---')
```

So we fall back to title sort (since all the dates will be the same)